src/pages/registration/RegistrationForm.tsx

```diff
@@ -278,15 +278,15 @@ const RegistrationForm = <T extends TissueValues<B>, B>({
                             )}
                           </FormikSelect>
                         )}
-                        {"solutionSample" in block && (
+                        {"solution" in block && (
                           <FormikSelect
                             emptyOption
                             label="Solution Sample"
```

Comment:
I think this should say "Solution"

Reply:
Anywhere else too that says "solution sample"

Reply:
Sorry, I didn't get that.

Reply:
Changes done 👍